test/vfd_swmr_common.h

```diff
@@ -88,6 +94,9 @@ H5TEST_DLL void esnprintf(char *, size_t, const char *, ...)
 
 H5TEST_DLL int fetch_env_ulong(const char *, unsigned long, unsigned long *);
 
+/* For demo */
+H5TEST_DLL size_t strlcpy(char *, const char *, size_t);
```

Comment:
OpenBSD puts strlcpy in string.h so this will cause problems on that platform. This should be renamed to h5_strlcpy or something like that.